rootfs/api/urls.py

```diff
@@ -106,7 +106,7 @@
     url(r'^auth/tokens/$',
         views.TokenManagementViewSet.as_view({'post': 'regenerate'})),
     # admin sharing
-    url(r'^admin/perms/(?P<username>[-_\w]+)/?$',
+    url(r'^admin/perms/(?P<username>.+)/?$',
```

Comment:
Although I guess that regex I pointed to would have prevented the creation of an illegal `username` in the first place, and at least this PR makes Workflow's own code more consistent...